modules/aruco/perf/perf_aruco.cpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#include "perf_precomp.hpp"
+
+namespace opencv_test {
+using namespace perf;
+
+// useAruco3Detection
+// CORNER_REFINE_SUBPIX
+typedef tuple <bool, int> ArucoTestParams;
+typedef TestBaseWithParam<ArucoTestParams> EstimateAruco;
+#define ESTIMATE_PARAMS Combine(Values(false, true), Values(-1))
+
+PERF_TEST_P(EstimateAruco, ArucoFirst, ESTIMATE_PARAMS )
+{
+    string imgPath = cvtest::findDataFile("gboriginal.png", false);
+    Mat image = imread(imgPath);
+    string dictPath = cvtest::findDataFile("tutorial_dict.yml", false);
+    cv::Ptr<cv::aruco::Dictionary> dictionary;
+
+    FileStorage fs(dictPath, FileStorage::READ);
+    aruco::Dictionary::readDictionary(fs.root(), dictionary); // set marker from tutorial_dict.yml
```

Comment:
Please add assertion that dictionary is read correctly and is not empty.

Reply:
added